src/RealtimeEffectPanel.cpp

```diff
@@ -138,52 +139,6 @@ namespace
       }
    };
 
-   //Alias for ListNavigationEnabled<wxWindow> which provides wxWidgets-style ctor
```

Comment:
This comment got lost in the cut and paste.

Reply:
And if restored, it should also be more complete, describing what the event handlers accomplish.